t/post.t

```diff
@@ -48,9 +48,6 @@ note("Not logged in - init");
     # user
     ok( !$vars->{remote} );
 
-    # icon
```

Comment:
Icons were removed by @momijizukamori but the test still expected them, so just removed that.